src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery ballSupplier = new Lottery();
```

Comment:
no need to make things harder to understand
```suggestion
        Lottery lottery = new Lottery();
```